.storybook/storybook.scss

```diff
@@ -183,7 +183,7 @@
   background: $interactive-01;
 }
 
-​ .color__brand-01 {
+​.color__brand-01 {
```

Comment:
Is there a css variable we can use instead?

Reply:
yes